### PR TITLE
refactor: Refactor createHearingBookings method (FPLA-1108)

### PR DESCRIPTION
**Pull request by muyinatech**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/FPLA-1108

### Change description ###

Refactor CaseDataGeneratorHelper.createHearingBookings() method.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: service/src/test/java/uk/gov/hmcts/reform/fpl/utils/EmailNotificationHelperTest.java [hunk 1]
```diff
@@ -101,7 +100,7 @@ void subjectLineShouldBeSuffixedWithHearingDate() {
         final LocalDateTime dateInTenMonths = LocalDateTime.now().plusMonths(10);
         CaseData caseData = CaseData.builder()
             .respondents1(createRespondents())
-            .hearingDetails(createHearingBookings(dateInTenMonths))
+            .hearingDetails(CaseDataGeneratorHelper.createHearingBookingsFromInitialDate(dateInTenMonths))
```
Comment (Gogetter) [inline service/src/test/java/uk/gov/hmcts/reform/fpl/utils/EmailNotificationHelperTest.java:103]: just for consistency, could we have this as static imports?

File: service/src/main/java/uk/gov/hmcts/reform/fpl/utils/CaseDataGeneratorHelper.java [hunk 2]
```diff
@@ -257,24 +236,18 @@ public static Others createOthers() {
...
-                            .country("UK")
-                            .build())
+                        .address(someAddress())
                         .build())
                     .build()
             )).build();
     }
 
     public static List<Element<GeneratedOrder>> createOrders(DocumentReference lastOrderDocumentReference) {
+        final String orderType = "Blank order (C21)";
```
Comment (Gogetter) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/utils/CaseDataGeneratorHelper.java:246]: 👍

File: service/src/main/java/uk/gov/hmcts/reform/fpl/utils/CaseDataGeneratorHelper.java [hunk 3]
```diff
@@ -585,4 +558,15 @@ private static CaseManagementOrder createApprovedCMO() {
                 .build())
             .build();
     }
+
+    private static Address someAddress() {
```
[the remark this answers is not in the document]
Comment (muyinatech) [reply service/src/main/java/uk/gov/hmcts/reform/fpl/utils/CaseDataGeneratorHelper.java:562]: Yes, I wil change.

File: service/src/test/java/uk/gov/hmcts/reform/fpl/utils/EmailNotificationHelperTest.java (hunk 1 above)
Comment (muyinatech) [reply service/src/test/java/uk/gov/hmcts/reform/fpl/utils/EmailNotificationHelperTest.java:103]: I will fix.

**Pull request merged**